### PR TITLE
fixup! add new section about generated signatures for bitfields

### PR DESCRIPTION
**Pull request by lkolbly**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -284,7 +284,7 @@
 //! |:--|:--|
 //! | `fn new() -> Self` | Creates a new instance of the bitfield with all bits initialized to 0. |
 //! | `fn from_bytes([u8; 1]) -> Self` | Creates a new instance of the bitfield from the given raw bytes. |
-//! | `fn as_bytes(&self) -> &[u8; 1]` | Returns the underlying bytes of the bitfield. |
+//! | `fn into_bytes(&self) -> &[u8; 1]` | Returns the underlying bytes of the bitfield. |
```
Comment (Robbepop) [inline src/lib.rs:287]:
```suggestion
//! | `fn into_bytes(self) -> [u8; 1]` | Returns the underlying bytes of the bitfield. |
```

Comment (Robbepop) [reply src/lib.rs:287]: This is the correct signature, now. :)

**Review by Robbepop — CHANGES_REQUESTED** [review]
Thanks a lot for this fix!
Glad you caught it. :)
I added one suggestion to your fix before we can merge.

File: src/lib.rs (hunk 1 above)
Comment (lkolbly) [reply src/lib.rs]: haha, that's good to know :P

**Review by Robbepop — APPROVED** [review]
Thanks for keeping the docs up to date! :)

**Comment by lkolbly** [comment]
Anytime!

**Pull request merged**